src/styles/dark-theme.less

```diff
@@ -1,15 +0,0 @@
-// dark-theme.less
-
-// Default imports
-@import '~antd/lib/style/color/colorPalette.less';
-@import '~antd/dist/antd.less';
-@import '~antd/lib/style/themes/dark.less';
-@import 'default-vars.less';
-
-// Variable overrides for dark mode specific styles
```

Comment:
After removing this file, how do we override certain components for dark mode? Let's keep a placeholder for overrides as I am sure we need to override some components sooner or later.